CMakeLists.txt

```diff
@@ -51,9 +59,29 @@ include(cmake/Dependencies.cmake)
 
 # Use project root as default include directory
 include_directories(${PROJECT_SOURCE_DIR})
+include_directories(${PROJECT_BINARY_DIR})
 
 # Compiler flags
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -fPIC")
 
 # Recurse into main project directory
 add_subdirectory(gloo)
+
+
+# Finally, create the cmake configuration files.
+configure_file(
+    ${PROJECT_SOURCE_DIR}/cmake/GlooConfigVersion.cmake.in
+    ${PROJECT_BINARY_DIR}/cmake/GlooConfigVersion.cmake
+    @ONLY)
+configure_file(
+    ${PROJECT_SOURCE_DIR}/cmake/GlooConfig.cmake.in
+    ${PROJECT_BINARY_DIR}/cmake/GlooConfig.cmake
+    @ONLY)
+install(FILES
+    ${PROJECT_BINARY_DIR}/cmake/GlooConfig.cmake
+    ${PROJECT_BINARY_DIR}/cmake/GlooConfigVersion.cmake
+    DESTINATION lib/cmake/Gloo
```

Comment:
I think sticking them in `<prefix>/share/cmake` is a better reflection of their purpose, since they are not libs. Per https://cmake.org/cmake/help/v3.0/command/find_package.html#command:find_package `<prefix>/share/cmake` is searched by default by find_package, so it won't matter for functionality.

Reply:
Sounds perfect.